packages/@lightningjs/ui-components/src/components/TabBar/index.js

```diff
@@ -16,4 +16,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-export { default as default } from './TabBar.js';
+import TabBar from './TabBar';
+import Tab from './Tab';
+
+export { TabBar as default, Tab };
```

Comment:
TabBar.js also exports Tab. That can probably be removed with this change, but also won't break anything if it remains there too.

Reply:
I think it'd be best to remove it, this is how we wind up with weird circular dependencies 😬

Reply:
Oh I didn't realize that 🤔 it seems a little odd to me. Thoughts on removing that and just having it live here in the index file?

Reply:
I think it makes the most sense to remove the Tab export from TabBar and have index.js handle all the exports of the module